tests/Tokenizer/Transformer/NameQualifiedTransformerTest.php

```diff
@@ -108,50 +108,52 @@ public static function provideProcessCases(): iterable
             ];
         }
 
-        yield 'name fully qualified 1' => [
-            [
-                new Token([T_OPEN_TAG, "<?php\n"]),
-                new Token([T_NS_SEPARATOR, '\\']),
-                new Token([T_STRING, 'Foo']),
-                new Token(';'),
-            ],
-            [
-                new Token([T_OPEN_TAG, "<?php\n"]),
-                new Token([T_NAME_FULLY_QUALIFIED, '\Foo']),
-                new Token(';'),
-            ],
-        ];
+        if (\defined('T_NAME_FULLY_QUALIFIED')) { // @TODO: drop condition when PHP 8.0+ is required
```

Comment:
https://github.com/sebastianbergmann/phpunit/issues/5659

Reply:
Too bad we need to double-check this, as there's `@requires PHP 8.0` already 😕. Thanks for spotting this.